blocks/logos/logos.css

```diff
@@ -0,0 +1,8 @@
+.slider-base {
```

Comment:
can you `. logos` here as this is already on the block, no need for extra `.slider-base`